api/proto/blockchain.proto

```diff
@@ -1,38 +1,13 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
 
-// Blockchain-related data types and service descriptions.
+// Blockchain-related data types.
 
 syntax = "proto3";
 import "google/protobuf/empty.proto";
 import "external.proto";
 
 package blockchain;
 
-service BlockchainAPI {
```

Comment:
Service-specific stuff now lives inside `consensus_common.proto` (these are API endpoints available to both peers and clients).